composer.json

```diff
@@ -61,7 +61,8 @@
     },
     "minimum-stability": "dev",
     "require": {
-        "oat-sa/lib-tao-qti": "~5.0.0",
+        "oat-sa/lib-tao-qti": "dev-fix/UDI-26-allow-minor-dependencies-upgrades as 5.0.1",
+        "qtism/qtism": "fix/UDI-26-polygon-intersection as 0.19.1",
```

Comment:
It's here temporarily.
Will update to a stable version of `oat-sa/lib-tao-qti` once released, removing a direct `qtism/qtism` dependency.